### ISSUE TITLE
Feature request--implement way of representing staff names in a multistaff part

### ISSUE THREAD

**Issue by hhpmusic** [w3c/musicxml#485]
As Sibelius released the December version last year, it included a feature of staff name, which can allow us to put a global part name at left, then individual players' names to every staff of the part. But Musicxml only has part-name and part-abbreviation, so I hope to include something like staff-name and staff-abbreviation in v4.1. This will enable complete export of the data, and it will benefit braille music translation a lot.

**Comment by adrianholovaty** [comment]
Just noting this is already something we've addressed in MNX, via the layouts system (e.g., [staff layout](https://w3c.github.io/mnx/docs/mnx-reference/elements/staff-layout/)).

**Comment by mscuthbert** [comment]
yes -- this is something that has long been missing in the representation system for MusicXML that I've noticed.  The question is -- is it something that can also change throughout the piece?  Like with individual instruments (Horn 3) moving among staves?  Or is it fixed from beginning (like "L.H." , "R.H.", "Ped." on certain organ scores)?

Supporting Braille better is always a compelling use case.

**Comment by hhpmusic** [comment]
It can be changed throughout the score, with instrument changes in Sibelius, where we input the same general part label and then apply different staff names. So when there's a change, both part- and staff-name/abbreviation tags are present. If there's a software which is able to change individual staff name without instrument changes, then part-name/abbereviation is not needed unless there's a new instrument change.
